resources/views/community/components/forum/post-comment-meta.blade.php

```diff
@@ -41,5 +41,14 @@
 
 <p class='smalltext !leading-[14px]'>
     {{-- Keep this all on a single line so white space isn't added before the comma --}}
-    {{ $formattedPostTimestamp }}@if($formattedEditTimestamp)<span class='italic smalltext !leading-[14px]'>, <span class='hidden sm:inline'>last</span> edited {{ $formattedEditTimestamp }}</span> @endif
+    <span title="{{ $postCreatedTimestamp }}" class="cursor-help">{{ $formattedPostTimestamp }}@if($formattedEditTimestamp), @endif</span>
+
+    @if($formattedEditTimestamp)
+        <span class='italic smalltext !leading-[14px]'>
+            <span class='hidden sm:inline'>last</span> edited
+            <span class="cursor-help" title="{{ $postEditedTimestamp }}">
+                {{ $formattedEditTimestamp }}
```

Comment:
Doesn't really make sense to have the helper tooltip for raw dates:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/6f3908f5-372f-4389-a9cf-ef83ec1c5ab6)

Reply:
Took a bit of conditional logic, but should be resolved in https://github.com/RetroAchievements/RAWeb/pull/1652/commits/1c9522c294615d9fff468fecfb064c704398e134.